.github/workflows/workflows.yaml

```diff
@@ -311,6 +311,174 @@ jobs:
           manifest-path: "flatpak/web/org.nanuc.Axolotl.yml"
           branch: "main"
 
+  build-axolotl-deb-arm64:
+    name: Build axolotl Debian arm64
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Go 1.16
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
```

Comment:
Lets extract the go and node version to global environment variables: else we duplicate this config for `build-axolotl` and `build-axolotl-deb-arm64`, for `build-axolotl-web` and `build-axolotl-web-deb-arm64`.

From what I can tell, it should be fairly straight-forward:

```yaml
env:
  GO_VERSION: 1.16
  NODE_VERSION: "16.x"
```

```yaml
with:
          node-version: $NODE_VERSION
```

Reply:
Shouldn't it be `${{ env.NODE_VERSION }}` then?

Reply:
try it out! I was looking at the examples here for my previous suggestion:

https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
I added the global variable in the beginning and used ${{ env.NODE_VERSION }} and ${{ env.GO_VERSION }} to recall the values.